frontend/src/lib/search.ts

```diff
@@ -85,7 +85,25 @@ export function queryRemoteOCPAppResources(): IRequestResult<ISearchResult> {
                             property: 'kind',
                             values: ['cronjob', 'daemonset', 'deployment', 'deploymentconfig', 'job', 'statefulset'],
                         },
-                        { property: 'cluster', values: ['!local-cluster'] },
+                    ],
+                },
+            ],
+        },
+        query: 'query searchResult($input: [SearchInput]) {\n  searchResult: search(input: $input) {\n    items\n  }\n}',
+    })
+}
+
+export function queryKustomizations(): IRequestResult<ISearchResult> {
+    return postRequest<SearchQuery, ISearchResult>(getBackendUrl() + apiSearchUrl, {
+        operationName: 'searchResult',
+        variables: {
+            input: [
+                {
+                    filters: [
+                        {
+                            property: 'kind',
+                            values: 'kustomization',
+                        },
```

Comment:
I wonder if it's possible to filter label by its key, tried with regex /xxxx=*/ but failed... not sure if that's supported. For now I used filter in later code to filter resources with specific labels

Reply:
Ah I see, thanks Feng! Guess we will have to it manually later then